### PR TITLE
Create waitForTimeout util

### PR DESCRIPTION
**Pull request by leroykorterink**
#119

### CODE REVIEW

File: src/utils/waitForTimeout/waitForTimeout.ts [hunk 1]
```diff
@@ -0,0 +1,9 @@
+/**
+ * Returns a promise that resolves after given amount of time. Use the default
+ * value of `0` to wait for the next tick.
+ */
+export function waitForTimeout(ms = 0): Promise<void> {
```
Comment (ThaNarie) [inline src/utils/waitForTimeout/waitForTimeout.ts:5]: I'm unsure myself what the best name could be here.

Strictly speaking, this function doesn't wait for anything. It returns a promise that resolve in a certain amount of time, that the caller could choose to wait for.

Following that logic, you would end up with something like `createTimeoutPromise`, but that doesn't feel good either. Maybe just `createTimeout` is good enough?

**Comment by ThaNarie** [comment]
Is this missing storybook docs? It's meant to be used from the outside, people should know about it from the docs :)

File: src/utils/waitForTimeout/waitForTimeout.ts (hunk 1 above)
Comment (mmnathan) [reply src/utils/waitForTimeout/waitForTimeout.ts]: Usually I see this named `sleep` (a bit technical) or `delay`, the it reads as `await delay(500)`.

**Comment by leroykorterink** [comment]
> Is this missing storybook docs? It's meant to be used from the outside, people should know about it from the docs :)

Added docs now, not sure why I missed that 😅

**Pull request merged**